waku/waku_rln_relay/group_manager/on_chain/group_manager.nim

```diff
@@ -622,7 +623,7 @@ method init*(g: OnchainGroupManager): Future[void] {.async.} =
   g.rlnContract = some(rlnContract)
   g.registryContract = some(registryContract)
 
-  if g.keystorePath.isSome() and g.keystorePassword.isSome():
+  if g.keystorePath.isSome() and existsFile(g.keystorePath.get()) and g.keystorePassword.isSome():
```

Comment:
we probably want to have a specific error log when a path and password is provided but it doesn't exist, what do you think?

Reply:
Sounds good! Added it now :)) lmk what you think about the new version